src/chrome/content/scaffold/templates/scrapeMARC.js

```diff
@@ -11,18 +11,18 @@ function scrape(doc, url) {
 		// ignore the table headings in lines[0]
 		record.leader = text(lines[1], 'td', 4);
 		var fieldTag, indicators, fieldContent;
-		for (let j=2; j<lines.length; j++) {
+		for (let line of Array.from(lines).slice(2)) {
```

Comment:
Is `Array.from` needed here?

Reply:
It is, as `querySelectorAll` yields a `NodeList` which can be converted into an array but is not actually an array.

Reply:
It's not needed. A `NodeList` is iterable, so `for…of` works.

Reply:
I stand corrected. In this case however it may still be needed; I'm looking at https://developer.mozilla.org/en-US/docs/Web/API/NodeList and it doesn't look like NodeList has `slice`.

Reply:
That's right. Sorry, I didn't notice that there were two separate lines here and one used `slice`.